src/components/common/blocks/gas-price/index.js

```diff
@@ -3,7 +3,12 @@ import PropTypes from 'prop-types';
 import _ from 'lodash';
 import web3Utils from 'web3-utils';
 
+import lightTheme from '@digix/gov-ui/theme/light';
```

Comment:
Oh, you don't have to pass this to the styled components. All styled components already have the `theme` props by default

Reply:
@mandres-digix - Actually, I need to pass this as well as the individual prop of `theme={this.props.theme}` to each and every `styled-component`, because `GasPrice` is being rendered in `governance-ui` whose `ThemeProvider` is of `Material-UI` which doesn't have all the values of `lightTheme`.

Let me know if you know of a better approach to tackle this. Also it begs the question on how can I tackle the same styles using themes, in case I move the `GasPrice` component to `governance-ui`? Thanks!

Reply:
> Actually, I need to pass this as well as the individual prop of `theme={this.props.theme}` to each and every `styled-component`, because `GasPrice` is being rendered in `governance-ui` whose `ThemeProvider` is of `Material-UI` which doesn't have all the values of `lightTheme`.

Ah, you're right. The `Material-UI` theme slipped my mind. In that case, I have no further comments for this PR. :+1:

> Also it begs the question on how can I tackle the same styles using themes, in case I move the `GasPrice` component to `governance-ui`? Thanks!

I don't think you can use the same themes since the two repositories are set up differently. As you can see in the code, `governance-ui` is still just putting the styles inline or via classes (it's a pretty old codebase). For `governance-ui` you might just have to do away with themes and revert to the old way of doing CSS.

Reply:
I thought so too. Hence I've setup the way it currently is for `GasPrice`. Took me long enough to figure that out honestly 😅